src/Tasks/Microsoft.NET.Build.Tasks.NETFramework/Microsoft.NET.Build.Tasks.NETFramework.csproj

```diff
@@ -0,0 +1,111 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <PackageId>Microsoft.NET.Sdk</PackageId>
+    <Description>The MSBuild targets and properties for building .NET Core projects.</Description>
```

Comment:
This Description isn't accurate.  A lot of the metadata can probably be deleted, and I don't think we need to multi-target this project.